test-site/src/index.ts

```diff
@@ -1,19 +1,105 @@
-import {analytics} from '@yext/analytics';
+import { analytics } from '@yext/analytics';
```

Comment:
One last thing, can we add a comment describing the weird behavior we saw in firefox
https://yext.slack.com/archives/G8UMH43HP/p1693419787243609?thread_ts=1693411231.167479&cid=G8UMH43HP